### PR TITLE
Adjust coding style 1

### LINKED ISSUE #1: Improve code quality
**Is your feature request related to a problem? Please describe.**
Currently our 2020v2 coding guidelines may not apply to the entire project.

**Describe the solution you'd like**
- Add `//#region` and `//#endregion` tags to every file
- Check for new-line issues
- Update the CONTRIB file with our 2020v2 coding guidelines
- Update all Licensing to `2020 Christian Aichner`

**Additional context**
Use https://github.com/snek-at/tonic/blob/master/STYLE_GUIDE.md as guide to update the CONTRIB file.

### PR DESCRIPTION
**Pull request by schettn**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently the coding style is not adapted to the 2020v2 standards. #1
- Currently anchors are not defined.

**What is the new behavior (if this is a feature change)?**
- Now the coding style is adapted to the 2020v2 standards. #1

**Other information**:
- 💮

### CODE REVIEW

**Review by Aichnerc — APPROVED** [review]
LGTM.

**Comment by schettn** [comment]
Anchors will be added in an upcoming coding style adjustment. So please have mercy.

**Review by sweggerat — APPROVED** [review]
looks good to me

File: src/components/pages/MessagePage/index.jsx [hunk 1]
```diff
@@ -1,109 +1,116 @@
...
+const ABOUT_PLACEHOLDER = {
   company: "Company",
   phone: "Company phone",
   address: "Company address",
   zip: "ZIP",
   city: "City",
-  mail: "mail@mail.com"
-}
+  mail: "mail@mail.com",
+};
```
Comment (kleberbaum) [inline src/components/pages/MessagePage/index.jsx:24]: Please add a ```//#endregion```.

File: src/components/pages/MessagePage/index.jsx [hunk 2]
```diff
@@ -1,109 +1,116 @@
...
       </MDBContainer>
     );
   }
 }
+//#endregion
 
+//#region > Exports
 export default MessagePage;
-
+//#endregion
```
Comment (kleberbaum) [inline src/components/pages/MessagePage/index.jsx:113]: Please add a newline.

File: src/store/reducers/rootReducer.js [hunk 3]
```diff
@@ -10,16 +11,20 @@ import { firestoreReducer } from "redux-firestore";
 
 //> Firebase reducer
 import { firebaseReducer } from "react-redux-firebase";
+//#endregion
 
+//> Constant Variables
```
Comment (kleberbaum) [inline src/store/reducers/rootReducer.js]: Mhhh there should be a region for that too.

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by LucaAllmaier — APPROVED** [review]
Looks good.

File: src/store/reducers/rootReducer.js [hunk 4]
```diff
@@ -10,16 +11,22 @@ import { firestoreReducer } from "redux-firestore";
 
 //> Firebase reducer
 import { firebaseReducer } from "react-redux-firebase";
+//#endregion
 
+//#region > Combined Reducers
```
Comment (kleberbaum) [inline src/store/reducers/rootReducer.js]: I think something more generally like Constants would fit better.

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**